lambdas/es/indexer/index.py

```diff
@@ -262,15 +264,20 @@ def handler(event, context):
                     )
                     continue
 
-                text = get_contents(
-                    bucket,
-                    key,
-                    ext,
-                    etag=etag,
-                    version_id=version_id,
-                    s3_client=s3_client,
-                    size=size
-                )
+                try:
+                    text = get_contents(
```

Comment:
Looks to me that if `get_contents` throws an exception (which will be caught), text will keep is previous value and we'll end up adding it twice. I think you want to `continue` here or maybe cleaner, move more of the logic into the try block.

Reply:
Per discussion: we won't add the document twice. We actually want files for which `get_contents()` fails to have an entry in ES. At the same time, the same files will end up in the DLQ when we re-raise at the very end of `handler()`.

I moved `text = ""` to the `except` clause so that it's easier to read.